src/main.ts

```diff
@@ -13,7 +13,10 @@ async function run(): Promise<void> {
   await setupRTX()
   await setEnvVars()
   await exec.exec('rtx', ['--version'])
-  await exec.exec('rtx', ['install'])
+  const install = core.getInput('install', {required: false})
+  if (install === 'true') {
```

Comment:
If I'm reading the docs right, the default is a "string representing the value", which means the default would be `true` not `'true'`. Either way, I would just make this check for truthiness so it works regardless
```suggestion
  if (install) {
```

Reply:
I applied your suggestion. Thank you.